src/main/java/org/folio/service/finance/FundService.java

```diff
@@ -45,24 +45,6 @@ public Future<List<Fund>> getAllFunds(Collection<String> fundIds, RequestContext
                 .collect(Collectors.toList()));
   }
 
-  public Future<List<Fund>> getFunds(Collection<String> fundIds, RequestContext requestContext) {
```

Comment:
deleted because there is almost identical method but with additional validation in this class